Failed Build

The app built successfully locally, but failed on Vercel with the following log:

```
Failed to compile.
./src/app/hero.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[13].oneOf[10].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[13].oneOf[10].use[3]!./src/app/hero.css
Error: Cannot find module '@tailwindcss/postcss'
Require stack:
- /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js
- /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/index.js
- /vercel/path0/node_modules/next/dist/build/webpack/config/index.js
- /vercel/path0/node_modules/next/dist/build/webpack-config.js
- /vercel/path0/node_modules/next/dist/build/webpack-build/impl.js
- /vercel/path0/node_modules/next/dist/compiled/jest-worker/processChild.js
    at Function.<anonymous> (node:internal/modules/cjs/loader:1225:15)
    at /vercel/path0/node_modules/next/dist/server/require-hook.js:55:36
    at Function.resolve (node:internal/modules/helpers:146:19)
    at loadPlugin (/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:49:32)
    at /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:157:56
    at Array.map (<anonymous>)
    at getPostCssPlugins (/vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/plugins.js:157:47)
    at async /vercel/path0/node_modules/next/dist/build/webpack/config/blocks/css/index.js:124:36
    at async /vercel/path0/node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js:52:40
    at async Span.traceAsyncFn (/vercel/path0/node_modules/next/dist/trace/trace.js:153:20)
Import trace for requested module:
./src/app/hero.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[13].oneOf[10].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[13].oneOf[10].use[3]!./src/app/hero.css
./src/app/hero.css
./src/pages/landing-page.tsx
./src/styles/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[13].oneOf[10].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[13].oneOf[10].use[3]!./src/styles/globals.css
Error: Cannot find module '@tailwindcss/postcss'
```


Basically, Cannot find module '@tailwindcss/postcss'

--- a/package.json
+++ b/package.json
@@ -33,12 +33,13 @@
   },
   "devDependencies": {
     "@eslint/eslintrc": "^3",
+    "@tailwindcss/postcss": "^4.1.4",
     "@types/node": "^20",
     "@types/react": "^19",
     "@types/react-dom": "^19",
     "eslint": "^9",
     "eslint-config-next": "15.1.7",
-    "postcss": "^8",
+    "postcss": "^8.5.3",
     "tailwindcss": "^4.1.4",
     "typescript": "^5"
   }

--- a/postcss.config.mjs
+++ b/postcss.config.mjs
@@ -1,7 +1,8 @@
 /** @type {import('postcss-load-config').Config} */
 const config = {
   plugins: {
-    '@tailwindcss/postcss': {},
+    tailwindcss: {},
+    autoprefixer: {},
   },
 };
 

--- a/postcss.config.mjs
+++ b/postcss.config.mjs
@@ -1,9 +1,7 @@
 /** @type {import('postcss-load-config').Config} */
 const config = {
   plugins: {
-    tailwindcss: {},
-    autoprefixer: {},
-  },
+    '@tailwindcss/postcss': {},  },
 };
 
 export default config;

--- a/src/components/hero/Hero.tsx
+++ b/src/components/hero/Hero.tsx
@@ -65,9 +65,9 @@ export default function Hero() {
                 Get your ticket
                 <ChevronRight className="w-6 h-6 ml-2 group-hover:translate-x-1 transition-transform" />
               </Link>
-              <Link href={paths.callForSpeakers()} className="border-2 border-white hover:border-white text-white px-8 py-4 rounded-full font-bold text-lg transition-all hover:bg-white/20 backdrop-blur-xs">
+              <a href={paths.callForSpeakers()} className="border-2 border-white hover:border-white text-white px-8 py-4 rounded-full font-bold text-lg transition-all hover:bg-white/20 backdrop-blur-xs">
                 Submit your talks
-              </Link>
+              </a>
             </div>
           </div>
 

